### ISSUE TITLE
dynamic hand gesture recognition

### ISSUE THREAD

**Issue by shizishen** [mdylan2/gesture_jester#10]
Can this my_model_weights.h5 model be trained for dynamic gesture recognition to recognize dynamic gestures?

thank you advance

**Comment by shizishen** [comment]
I am a first time to learn deeplearning

**Comment by mdylan2** [comment]
Hi, currently the model only supports fixed gestures, since the Convolutional Neural Network accepts fixed inputs as images and uses Categorical Cross Entropy as its loss function. 

To solve your issue, off the top of my head, you could modify the Conv Net to accept video input and then do a categorical prediction to predict the dynamic gesture. 

However, implementation of this feature is out of the scope of this project.

Hope the advice above helps though!

**Comment by mdylan2** [comment]
Hi @shizishen. 

The Conv Net currently only supports static images so dynamic gestures are not possible at this point.

However, off the top of my head, you could modify the CNN architecture to accept video as input and predict a gesture from that, which would allow you to create the feature you need.

Unfortunately, this is out of the scope of this project but hope that my advice helps!
